### ISSUE TITLE
Need traces on errors within SOAP messages

### ISSUE THREAD

**Issue by SergeVil** [krasserm/ipf#88]
Please, provide the traces when the problem occurred within the code. 

javax.xml.bind.annotation.XmlEnum.ErrorCode contains the list of errors that appear at the SOAP response.

When I am getting for example REGISTRY_METADATA_ERROR, I need after that to go and modify the other class to see what exactly happened and restart my test:

public class Iti42Service extends AbstractWebService implements Iti42PortType {
    @Override
    public RegistryResponseType documentRegistryRegisterDocumentSetB(SubmitObjectsRequest body) {
        Exchange result = process(body);
        if (result.getException() != null) {

```
            // My code starts
        System.out.println("Iti42Service: " + result.getException().getMessage());
        result.getException().printStackTrace();
            // My code ends

        Response errorResponse = new Response(result.getException(), ErrorCode.REGISTRY_METADATA_ERROR, ErrorCode.REGISTRY_ERROR, null);
        return EbXML30Converters.convert(errorResponse);
    }

    return Exchanges.resultMessage(result).getBody(RegistryResponseType.class);            
}
```

} 

I would like to track the problems just by using log4j configurations.

Thank you,
Serge

**Comment by unixoid** [comment]
The requested functionality will be available in IPF 2.4-m2.
What meaning is the enumeration javax.xml.bind.annotation.XmlEnum.ErrorCode supposed to have in this context?

**Comment by unixoid** [comment]
The requested functionality will be available in IPF 2.4-m2.
What meaning is the enumeration javax.xml.bind.annotation.XmlEnum.ErrorCode supposed to have in this context?

**Comment by unixoid** [comment]
The requested functionality will be available in IPF 2.4-m2.
What meaning is the enumeration javax.xml.bind.annotation.XmlEnum.ErrorCode supposed to have in this context?
